content/tutorials/react-native-presence-publish-subscribe.textile

```diff
@@ -1,155 +1,152 @@
+# publish: We will cover the concept of publishing messages in Ably
+# subscribe: We will cover the concept of subscribing to channels and events in Ably
+# react-native: We will build the chat application using @React-Native@ 
 
 Note: You can give your project any other name you wish to.
 
 Wait until React Native does all its installations, then you can change directory into the new project and run the application:
 
 ```[sh]
-//change directory to ably-presence-publish-subscribe-react-native
-cd ably-presence-publish-subscribe-react-native
-//run the application for android
+#change directory to Ably_presence_publish_subscribe_react_native
+cd Ably_presence_publish_subscribe_react_native
+#run the application for android
 react-native run-android
-//run the application for ios
+#run the application for ios
 react-native run-ios
 ```
+"See this step in Github":https://github.com/samuelayo/tutorials/commit/2b52467070a6dd6ae999ce59e788dff9eac8a84b
 
 If all goes well, you should see a screen that says “Welcome to React Native” come up on your emulator or device.
 
-
-h2. Setting Up Ably
-
-You need to sign up on "Ably":https://www.ably.io/, create a new app, click on the @manage app@ button at the left-hand side of the app, click on the @Api keys@  tab and copy our Api Key.
-
-<a href="/images/tutorials/copy-api-key.png" target="_blank">
-  <img src="/images/tutorials/copy-api-key.png" style="width: 100%" alt="Copy API Key screenshot">
-</a>
-
 Next, you will install the required libraries for this app. 
 
 ```[sh]
-npm install ably ably-react-native express body-parser 
+npm install ably ably-react-native express body-parser --save
 ```
-In the above bash command, you installed 4 packages. I will explain what the four packages are used for:
+In the above bash command, you installed 4 packages. Here is the explanation of what the four packages are used for:
 
-# express: This is a Node.js web framework which we’ll use to create our API.
-# body-parser: This library is used by Express to parse body requests.
-# ably: This library is the official @Ably@ library for Node js
-# ably-react-native: This library is the official @Ably@ library for React Native
+# Express: This is a Node.js web framework which we’ll use to create our API.
+# Body-parser: This library is used by Express to parse body requests.
+# Ably: This library is the official @Ably@ library for Node js
+# Ably-react-native: This library is the official @Ably@ library for React Native
 
+"See this step in Github":https://github.com/samuelayo/tutorials/commit/b321b24c694cd9d3d67b9724132eb46da7fa1821
 
-h2. Creating Our API
+h2.
+  default: Step 3 - Creating Our API
 
 First, Create a new file called @server.js@, which serves as your API in our root folder and paste the following:
 
 ```[javascript]
 //require express
 var express = require('express')
-//define app as in instance of express
+    //define app as in instance of express
```

Comment:
What's this indentation for?  Same below.

Reply:
Sorry about this, fixed it now with the recent push.